internal/buf/bufimage/bufimagebuild/builder_test.go

```diff
@@ -314,3 +331,12 @@ func testGetImageImportPaths(image bufimage.Image) []string {
 	sort.Strings(importNames)
 	return importNames
 }
+
+func testFileAnnotations(t *testing.T, relDirPath string, wantMessages ...string) {
```

Comment:
This hit the 3 times threshold so I extracted it.

Reply:
We don't have a 3 times threshold, but this is fine